### ISSUE TITLE
Astronomisk romforskning

### ISSUE THREAD

**Issue by anjamaa** [realfagstermer/realfagstermer#874]
Kan denne slås sammen med `Astrofysisk romforskning`?

--
URI: http://data.ub.uio.no/realfagstermer/c009930
Bruk: http://ub-viz01.uio.no/okapi2/#/search?q=realfagstermer%3A%22Astronomisk%20romforskning%22
Soksed: https://soksed.biblionaut.net/concepts/REAL009930

**Comment by mittinatten** [comment]
Trenger en å skille mellom Astronomisk romforskning og Astronomi, og det samme for Astrofysikk?

**Comment by danmichaelo** [comment]
Det ser ut som [`Romforskning`](https://snl.no/romforskning) tilsvarer `Space exploration`? Men inndelingen av `Romforskning` i `Astronomisk romforskning` og `Astrofysisk romforskning` skjønner jeg ikke helt. @superLine ?

**Comment by Hypsibius** [comment]
Ikkje mitt felt, men er ikkje den logiske omsetjinga av (Space) exploration UTforskning og ikkje forskning?

**Comment by danmichaelo** [comment]
> Ikkje mitt felt, men er ikkje den logiske omsetjinga av (Space) exploration UTforskning og ikkje forskning?

Jo, det er den logiske oversettelsen, men ser ikke ut som den er i særlig bruk (ref. SNL, Wikipedia, "romutforskning site:uio.no")
